sdk/communication/Azure.Communication.Chat/tests.yml

```diff
@@ -0,0 +1,17 @@
+trigger: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: communication
+    Project: Azure.Communication.Chat
+    CloudConfig:
```

Comment:
@benbp do you think it is worth having a communication level template to capture some of this duplication?